docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -67,16 +52,46 @@ components:
           imageReference: eu.gcr.io/gardener-project/landscaper/examples/blueprints/guided-tour/external-blueprint:1.0.0
 ```
 
-If you were to prefer to embed the blueprint in the component as a local blob, instead of an `access:...`, you would have to
-specify an `input:...` as demonstrated [here](./config-files/local-blob-components.yaml). The commands used to create the actual component based on the *component configuration file* and to upload this
-component to an OCI registry can be found [here](./commands/component.sh).
+The commands used to create the actual component version based on the *component configuration file* and to upload this
+component version to an OCI registry can be found [here](./commands/component.sh). The first command creates
+a file system representation of the component version and the second uploads this to the registry:
 
->**Tip:** If you need the same resource in multiple components, instead of copy-pasting, you might want to use 
-> [*resource configuration files*](https://ocm.software/docs/guides/getting-started-with-ocm/#using-a-resources-file).
+```
+ocm add components --create --file ../tour-ctf ../config-files/components.yaml
+ocm transfer ctf --enforce ../tour-ctf eu.gcr.io/gardener-project/landscaper/examples
```

Comment:
Why the enforce flag?

Reply:
To overwrite an already existing version.